package.json

```
@@ -235,18 +235,17 @@
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.5",
     "@reduxjs/toolkit": "^1.9.7",
+    "@reown/walletkit": "^1.2.0",
```

```suggestion
    "@reown/walletkit": "^1.2.1",
```
new patch version is available for WalletKit that is using `2.18.1` `@walletconnect` deps. having all `@reown` and `@walletconnect` deps to latest will avoid having mismatched inner dependencies